packages/webapp/webapp_server.js

```diff
@@ -439,9 +440,9 @@ var getUrlPrefixForArch = function (arch) {
     '' : '/' + '__' + arch.replace(/^web\./, '');
 };
 
-// parse port to see if its a Windows Server style named pipe. If so, return as-is (String), otherwise return as Int
+// parse port to see if it's a Windows Server-style named pipe or a UNIX path/filename for a UNIX domain socket file. If so, return as-is (String), otherwise return as Int
 WebAppInternals.parsePort = function (port) {
-  if( /\\\\?.+\\pipe\\?.+/.test(port) ) {
+  if( /\\\\?.+\\pipe\\?.+/.test(port) || /^(.+)\/([^/]+)$/.test(port) ) {
```

Comment:
Could you update the [existing `parsePort` tests](https://github.com/meteor/meteor/blob/f0dd494c82244df79b544bdbf590ebecc0412189/packages/webapp/webapp_tests.js#L160) to verify this additional parsing works properly?